v3-nginx/Dockerfile-alpine

```diff
@@ -124,6 +124,9 @@ ENV ACCESSLOG=/var/log/nginx/access.log \
     MODSEC_UPLOAD_DIR=/tmp/modsecurity/upload \
     PORT=80 \
     NGINX_ALWAYS_TLS_REDIRECT=off \
+    SET_REAL_IP_FROM="127.0.0.1,10.10.0.0/24" \
```

Comment:
Why the `10.10.0.0/24` here?

Reply:
Good catch! I had that in there for debugging.